lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -23,7 +26,13 @@ $spec = @{
         proxy_username = @{ type='str' }
         proxy_password = @{ type='str'; no_log=$true }
         force = @{ type='bool'; default=$true }
+        checksum = @{ type='str'; aliases=@( 'hash' ) }
```

Comment:
I would rather not introduce a new alias here. It is not in the **get_url** module and I would prefer it not here either.
In most cases aliases are used to rename a parameter (for consistency) but keep the old one for backward compatibility.

Reply:
Fixed